test/node/connection-inspect.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+
+const test = require('tap');
+
+const jstp = require('../..');
+
+const app = require('../fixtures/application.js');
+const application = new jstp.Application(app.name, app.interfaces);
+
+let server;
+let client;
+
+test.beforeEach((done) => {
+  server = jstp.tcp.createServer(0, [application]);
+  server.listen(() => {
+    const port = server.address().port;
+    client = jstp.tcp.createClient({ host: 'localhost', port });
+    done();
+  });
+});
+
+test.afterEach((done) => {
+  client.disconnect();
+  server.close();
+  done();
+});
+
+test.test('must process inspect packets', (test) => {
+  const expectedInterfaces = Object.keys(app.interfaces);
+  let expectedTests = 1;
+  expectedInterfaces.forEach((iface) => {
+    expectedTests += 1;
```

Comment:
@nechaido, if that's the case, may you add a comment explaining this, to avoid confusion in the future?
Also, why not just add `expectedInterfaces.length` to `expectedTests` immediately after initialization, or even initialize `expectedTests` with  `expectedInterfaces.length + 1`?